source/_patterns/01-atoms/nav-item/demo/nav-items.yml

```diff
@@ -0,0 +1,7 @@
+nav:
```

Comment:
When we bring active over to Drupal, we'll need to account for `is-active`, which should be supplied by Drupal itself. It might be worth thinking about how we might manage this all at once.